javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,82 @@
+console.log("Hello"); //Initial code
+
+//Age-ify (A future age calculator)
+const yearOfBirth = 1989;
+const yearFuture = 2050;
+
+//with using - arithmatic operator we can calculate the age.
+let myAge  = yearFuture  - yearOfBirth;
+
+console.log("I will be " + myAge + " years old in " + yearFuture);
+
+
+//Goodboy-Oldboy (A dog age calculator)
+ const dogYearOfBirth = 2017;
+ const dogYearFuture = 2027;
+ const dogYear = dogYearFuture - dogYearOfBirth;
+ const dogHumunsYear = 7 * dogYear ;
```

Comment:
I think it is opposite 
```suggestion
 const dogYear = (dogYearFuture - dogYearOfBirth) * 7;
 const dogHumunsYear = dogYear / 7 ;
```

Reply:
Yes right oopppsss I  understood totally opposite , but i will change it, thanks